tests/aws/test_network_configuration.py

```diff
@@ -20,7 +20,7 @@
 from localstack.utils.files import new_tmp_file, save_file
 from localstack.utils.strings import short_uid
 
-pytestmark = [markers.parity.only_localstack]
+pytestmark = [markers.aws.only_localstack]
```

Comment:
i think for the next iteration we should try to get rid of this module-level markers as well :)